streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/graph/executiongraph/ExecutionGraph.java

```diff
@@ -0,0 +1,94 @@
+package org.ray.streaming.runtime.core.graph.executiongraph;
+
+import java.io.Serializable;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Physical plan.
+ */
+public class ExecutionGraph implements Serializable {
+
+  private final String jobName;
+  private List<ExecutionJobVertex> executionJobVertexList;
```

Comment:
`executionJobVertexList ` can be binded by `executionJobVertexMap.values()`.

Reply:
LGTM.